index.html

```diff
@@ -193,5 +193,64 @@ <h2>About me</h2>
         </li>
       </ul>
     </section>
+
+    <footer>
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/myyodjdo" method="post">
+        <input type="text" placeholder="Full name" name="Full name" maxlength="30" required />
+        <input type="email" placeholder="Email address" name="email" required />
+        <textarea
+          cols="30"
+          rows="10"
+          placeholder="Enter text here"
+          maxlength="500"
+          required
+          name="Message"
+        ></textarea>
+        <button type="reset">Get in touch</button>
```

Comment:
- [ ] You have done a great job with form structure but the `type` of the form submission button suppose to be `type="submit"` not `reset` because now the submission process is not working online.